Slider/Assets/Scripts/UI/Artifact/Screens/UIArtifactInventory.cs

```diff
@@ -23,7 +24,20 @@ public class UIArtifactInventory : MonoBehaviour
     private void OnEnable() 
     {
         UpdateIcons();
-        UpdateText("Collection");
+        if(Controls.CurrentControlScheme == Controls.CONTROL_SCHEME_CONTROLLER)
+        {
+            ArtifactInventoryCollectible s = GetLeftmostSelectible();
+            if(s != null)
+            {
+                s.controllerSelectible.Select();
+                s.controllerSelectionImage.enabled = true;
+                s.UpdateInventoryName();
+            }
+            else
+                UpdateText("Collection");
```

Comment:
do we want to get rid of this since it seems like the artifact screen animator is responsible for handling this? its duplicated otherwise